src/main/java/io/codurance/pouch/ResourceController.java

```diff
@@ -31,4 +29,10 @@
                 .map(ResponseEntity::ok)
                 .orElseGet(() -> ResponseEntity.status(NOT_FOUND).build());
     }
+
+    @PostMapping("/resources")
+    ResponseEntity<Resource> save(@RequestBody Resource input) {
```

Comment:
The intent expressed in the acceptance test is "should create one resource", so I'd expect the controller method to reflect this. How about `create` instead of `save`?

Reply:
Oh, I see where you're coming from with this. Well, all methods in my controller have exactly the same names as the corresponding methods in the default CRUD repository.
So far this is:
`findAll`
`findById`
`save`
While I think this will prevent confusion, I also see your point. I did paraphrase what's going on quite a bit with this one (add a resource, store resource in the database, create one resource, save resource). I will aim for more consistency. And I guess the controller should speak domain language, while the repository should use more technical terms - a layer of abstraction sitting between the two.